detection_rules/rule_validators.py

```diff
@@ -81,34 +97,44 @@ def validate(self, data: 'QueryRuleData', meta: RuleMeta) -> None:
             return
 
         indexes = data.index or []
-        beats_version = meta.beats_version or beats.get_max_version()
-        ecs_versions = meta.ecs_versions or [ecs.get_max_version()]
+        stack_versions = get_stack_schemas(meta.min_stack_version)
```

Comment:
do you think some of this is boilerplate now between the subclasses and could be a helper method on the base QueryValidator class?

Reply:
4c54015